File: tests/ai-fixtures/vibecheck-ai.java
public class F {
  void h(javax.servlet.http.HttpServletRequest req) {
    // ruleid: vibecheck-untrusted-input-in-system-prompt-java
    SystemMessage.from(req.getParameter("persona"));
  }
  void ex(Client client, Object params) throws Exception {
    var resp = client.messages().create(params);
    // ruleid: vibecheck-llm-output-executed-java
    Runtime.getRuntime().exec(resp.content());
  }
}

File: tests/ai-fixtures/vibecheck-ai.go
package main
import (
	"os/exec"
	"net/http"
)
func h(r *http.Request) {
	// ruleid: vibecheck-untrusted-input-in-system-prompt-go
	openai.SystemMessage(r.URL.Query().Get("persona"))
}
func ex(c *Client, ctx interface{}, params interface{}) {
	resp, _ := c.Messages.New(ctx, params)
	// ruleid: vibecheck-llm-output-executed-go
	exec.Command("sh", "-c", resp.Content)
}
